### DISCUSSION TITLE
Copilot Handoff - How to Use?

### DISCUSSION THREAD

**Discussion by lifan0127** [microsoft/teams-sdk#1742] [Q&A]
Hi @aacebo , I see you recently introduced the handoff handler (https://github.com/microsoft/teams-ai/pull/1562). But I can't figure out how to use it. I am interested in the bot-to-bot handoff feature as described in the following article, but their example doesn't use the `team-ai` library. Do you have any guidance?

https://learn.microsoft.com/en-us/microsoftteams/platform/bots/how-to/conversations/bot-copilot-handoff

https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/msgext-copilot-handoff/ts

**Comment by aacebo** [comment]
here is an example of using the handler 
![image](https://github.com/microsoft/teams-ai/assets/44811138/2ccdc85d-afa3-458d-8f1e-e9251a3c2aa3)
we don't have a sample for this yet but we will in the future.

**Reply by lifan0127** [reply]
Thanks for your prompt response! I still don't know how to trigger the handoff handler.  Here is what I tried:

- Add the handoff handler in Bot A
- In another bot (Bot B), display an Action.OpenUrl button with a "deep link" pointing to Bot A (and a random string as the continuation token)
- Click on the link, my Teams window switches to Bot A, but the handoff function is not triggered.

**Reply by lifan0127** [reply]
Hi @aacebo , is there any update regarding this feature? I imagine that, for many customers, Copilot interoperability is becoming an important, though not immediate, requirement for custom AI bots. Thanks!

**Comment by heyitsaamir** [comment]
We're clearing discussions and starting fresh. This discussion has not been acted upon. Please reopen the discussion, or start a new one if this question is still important. Thank you for your understanding.
